en/docs/learn/pushed-authorization-endpoint.md

```diff
@@ -0,0 +1,68 @@
+The pushed authorization request endpoint is an HTTP API at the authorization server that accepts HTTP "POST" requests with
```

Comment:
```suggestion
The Pushed Authorization Request endpoint is an HTTP API at the authorization server that accepts HTTP "POST" requests with
```

Reply:
The CDS spec use 'z'

Reply:
The first occurence of Pushed Auth Req we need to use the Abbreviation, otherwise we cannot use it throughout the doc.